velox/exec/SortWindowBuild.h

```diff
@@ -69,6 +69,9 @@ class SortWindowBuild : public WindowBuild {
   // structure that helps simplify the window function computations.
   void computePartitionStartRows();
 
+  // Find the nest partition start row from start.
```

Comment:
nest -> next

Returns the row where next partition starts.

Reply:
Done.